### DISCUSSION TITLE
Remove genes from Archr object

### DISCUSSION THREAD

**Discussion by ACAIDI** [GreenleafLab/ArchR#1901] [Questions / Documentation]
Hello,
I would like to know if it's possible to remove some genes from ArchR object. If it is possible how?
Thanks in advance.

**Comment by rcorces** [comment]
You need to remove them from the project's geneAnnotation. Depending on your downstream application, you may need to remove the corresponding entry in `genes` `exons` and `tss`. You also need to force re-addition of a new GeneScoreMatrix after tweaking your geneAnnotation.
`proj@geneAnnotation`
